From 5924e3ad82b5dfc95acf9f6751946d5f086d9ce7 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date: Tue, 1 Jan 2019 20:13:57 +0100
Subject: Drop unused dependencies

Test Plan: Built fine with deps removed.

Reviewers: #calligra:_3.0, danders

Reviewed By: #calligra:_3.0, danders

Subscribers: danders

Differential Revision: https://phabricator.kde.org/D17903
---
 CMakeLists.txt | 14 ++------------
 1 file changed, 2 insertions(+), 12 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3cb4b63..a6a3c79 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -121,7 +121,6 @@ find_package(KF5 ${REQUIRED_KF5_VERSION} REQUIRED
             IconThemes
             ItemViews
             JobWidgets
-            KCMUtils
             KIO
             Notifications
 #             NotifyConfig
@@ -169,22 +168,13 @@ find_package(Qt5 ${REQUIRED_QT_VERSION} REQUIRED
         COMPONENTS
             Core
             Gui
-            Network
             PrintSupport
-            Svg
             Test
             Widgets
             Xml
     )
-    find_package(Qt5 ${REQUIRED_QT_VERSION} QUIET
-        COMPONENTS
-            DBus
-            OpenGL
-    #        Quick
-    #        Sql
-    #         WebKit
-    #         WebKitWidgets
-    )
+    find_package(Qt5DBus ${REQUIRED_QT_VERSION} QUIET)
+
     # Qt5Declarative was removed in Qt 5.6.0 so search for it in a separate call
     # Including it in a collected find_package(Qt5 ...) call can lead to a fatal not-found error:
     # * Qt5 (required version >= 5.3.0)
-- 
cgit v1.1
